src/routes/v2/authenticated/review.ts

```diff
@@ -67,33 +70,80 @@ export class ReviewsRouter {
 
   createReviewRequest: RequestHandler<
     { siteName: string },
-    { reviewers: string[] },
-    unknown,
+    { pullRequestNumber: number } | ResponseErrorBody,
+    { reviewers: string[]; title: string; description: string },
     unknown,
     { userSessionData: UserSessionData }
   > = async (req, res) => {
-    // Step 1: Check that user exists.
+    // Step 1: Check that the site exists
+    const { siteName } = req.params
+    const site = await this.sitesService.getBySiteName(siteName)
+
+    if (!site) {
+      return res.status(404).send({
```

Comment:
actually this is a common scenario - if a user miskeys their repo name, this will happen as we don't validate `siteName` until this point!

that aside, i think this is a good suggestion and i'll probably do this for the remaining functions in this file also